build.gradle

```diff
@@ -152,12 +153,79 @@ tasks.whenTaskAdded {task ->
     }
 }
 
+def splitTestConfig = [
+    dlicDlsflsTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.dlic.dlsfls.*"
+        ]
+    ],
+    dlicRestApiTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.dlic.rest.*"
+        ]
+    ],
+    crossClusterTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.ccstest.*"
+        ]
+    ],
+    sslTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.ssl.*"
+        ]
+    ],
+    securityIntegrationTest: [
+        includeTestsMatching: [
+            "org.opensearch.security.*Integ*"
+        ],
+        excludeTestsMatching: [
+            "org.opensearch.security.sanity.tests.*",
+            "org.opensearch.security.ssl.OpenSSL*",
```

Comment:
I think they appeared here due to the `opensslTest` task

Reply:
Can we remove the other tasks likes `opensslTest` and consolidate in a single way to filter/execute them?

Reply:
@willyborankin Fixed.

@peternied I've removed `opensslTest` as all of its tests are included in `sslTest` task.

Reply:
I've restored openSSL tests because we don't know whether issues from #2301 are still affecting tests and I don't want to break existing workflows.

Reply:
We should remove the either `opensslTest` or `opensslCiTest`.  I would recommend only keeping `opensslCiTest` around

Reply:
@peternied Removed.